site/gatsby-site/src/utils/discoverIndexGenerator.js

```diff
@@ -1,167 +0,0 @@
-const { getUnixTime } = require('date-fns');
-
-module.exports = async ({ graphql }) => {
-  const { data } = await graphql(`
-    query AllIncidentsDataQuery {
-      allMongodbAiidprodIncidents {
-        nodes {
-          date
-          incident_id
-          reports
-        }
-      }
-
-      allMongodbAiidprodReports {
-        nodes {
-          mongodb_id
-          description
-          authors
-          image_url
-          cloudinary_id
-          language
-          source_domain
-          text
-          plain_text
-          title
-          url
-          date_downloaded
-          date_modified
-          date_published
-          date_submitted
-          epoch_date_downloaded
-          epoch_date_modified
-          epoch_date_published
-          epoch_date_submitted
-          submitters
-          date_submitted
-          report_number
-          ref_number
-        }
-      }
-
-      allMongodbAiidprodClassifications(
-        filter: { namespace: { eq: "CSET" }, classifications: { Publish: { eq: true } } }
-      ) {
-        nodes {
-          incident_id
-          namespace
-          classifications {
-            Harm_Distribution_Basis
```

Comment:
@smcgregor some keys are missing and not being pushed to Algolia, like `Annotator`, `Reviewer`, `Quality Control`, etc. should all of them be pushed or should I keep a whitelist?

Reply:
Those fields are internal process fields that should only be displayed to authenticated users. The purpose of defining the fields of the taxonomies in the DB is to support this sort of white/blacklisting.